examples/chart-conversion/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "local-mashup",
+  "version": "1.0.0",
+  "description": "Simple local mashup",
+  "main": "index.html",
+  "scripts": {
+    "start": "parcel index.html --open",
+    "build": "parcel build index.html"
+  },
+  "dependencies": {
+    "@nebula.js/sn-bar-chart": "latest",
+    "@nebula.js/sn-combo-chart": "^1.4.1",
+    "@nebula.js/sn-line-chart": "latest",
+    "@nebula.js/sn-pie-chart": "^1.9.1",
+    "@nebula.js/stardust": "latest",
+    "enigma.js": "2.7.2"
+  },
+  "devDependencies": {
+    "@babel/core": "7.12.3",
+    "parcel-bundler": "^1.12.4"
```

Comment:
parcel bundler is deprecated, we should avoid using it.
There are issues with the new parcel, but hopefully they can resolve later, so I think we should stick with it.

Reply:
Sure. gonna use the new parcel.

Reply:
Done